mods/ra/maps/siberian-conflict-1-fresh-tracks/fresh-tracks.lua

```diff
@@ -0,0 +1,282 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+TimerTicks = DateTime.Minutes(10)
+Squad1 = { FirstSquad1, FirstSquad2, FirstSquad3 }
+Squad2 = { SecondSquad1, SecondSquad2, SecondSquad3 }
+PatrolMammothPath = { Patrol1.Location, Patrol2.Location, Patrol3.Location, Patrol4.Location, Patrol5.Location, Patrol6.Location, Patrol7.Location }
+ConvoyEscape = { CPos.New(113, 42), CPos.New(117, 71) }
```

Comment:
Is this generally how this is done, or do we usually place `waypoint`s (and use their `Location`)?

Reply:
For cell triggers we usually do it this way.